packages/datetime/src/common/dateUtils.ts

```diff
@@ -110,6 +110,27 @@ export function isMonthInRange(date: Date, dateRange: DateRange) {
     return start <= day && day <= end;
 }
 
+export function isTimeInRange(time: Date, minTime: Date, maxTime: Date): boolean {
+    time = getDateOnlyWithTime(time);
+    minTime = getDateOnlyWithTime(minTime);
+    maxTime = getDateOnlyWithTime(maxTime);
```

Comment:
:x: _never reassign arguments;_ they should be treated as constants. modifying them is a serious side effect and can affect calling code. define new variables here.